types/relay-runtime/lib/store/RelayStoreTypes.d.ts

```diff
@@ -395,6 +395,67 @@ export interface RecordSourceSelectorProxy<T = {}> extends RecordSourceProxy {
     insertConnectionEdge_UNSTABLE(connectionID: ConnectionID, args: Variables, edge: RecordProxy): void;
 }
 
+interface LogEventQueryResourceFetch {
+    readonly name: 'queryresource.fetch';
+    readonly operation: unknown;
```

Comment:
Why is this `unknown`? Check [this](https://github.com/facebook/relay/blob/master/packages/relay-runtime/store/RelayStoreTypes.js#L401) and [this](https://github.com/facebook/relay/blob/master/packages/relay-runtime/store/RelayStoreTypes.js#L117-L121).